apps/system-e2e/src/support/index.ts

```diff
@@ -14,5 +14,10 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
+Cypress.Cookies.debug(true)
```

Comment:
Do we wanna keep the Cookies.debug in?
Wasn't it just beneficial while working on this PR (and will become noise to others)?

Reply:
It's really low volume, and might become beneficial when and if we need to debug failed logins.